wapi/src/main/java/com/mycelium/wapi/api/WapiClientElectrumX.kt

```diff
@@ -20,6 +21,7 @@ import com.mycelium.wapi.api.response.*
 import com.mycelium.wapi.model.TransactionOutputEx
 import com.mycelium.wapi.model.TransactionStatus
 import kotlinx.coroutines.CancellationException
+import sun.net.www.http.HttpClient
```

Comment:
import was added, but I can't see it is used in the new code in the file -> unused?

Reply:
right, removed